inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/format/JsonDynamicSchemaFormat.java

```diff
@@ -99,7 +99,7 @@ public abstract class JsonDynamicSchemaFormat extends AbstractDynamicSchemaForma
 
     protected JsonDynamicSchemaFormat() {
         this.rowDataConverters =
-                new JsonToRowDataConverters(true, false, TimestampFormat.ISO_8601);
+                new JsonToRowDataConverters(false, false, TimestampFormat.ISO_8601);
```

Comment:
suggested a configurable parameter

Reply:
it's recommanded to config TimestampFormat as a configurable parameter.